src/libs/kata-types/src/config/hypervisor/dragonball.rs

```diff
@@ -191,6 +192,15 @@ impl ConfigPlugin for DragonballConfig {
                     MIN_DRAGONBALL_MEMORY_SIZE_MB
                 ));
             }
+            if !db.memory_info.hugepages.is_empty()
```

Comment:
That test does not seem Dragonball-specific, frankly. If that's the only test being done, it can probably move in the config parsing code.